examples/generator-fullstack/.graphqlrc.yml

```diff
@@ -3,16 +3,12 @@ schema:
     require: ts-node/register
 documents: "./client/src/graphql/**/*.ts"
 extensions:
+  # Graphback configuration
   generate:
-    db:
-      dbConfig:
-        user: postgresql
-        password: postgres
-        database: users
-        host: localhost
-        port: 55432
-      database: pg
-    graphqlCRUD:
+    ##  Input schema
+    model: ./model
+    ## Global configuration for CRUD generator
+    crudSpec:
```

Comment:
Hmm, `crudConfig`? Not sure if this is a spec.

Reply:
maybe crudOptions?